src/main/kotlin/skytils/skytilsmod/core/Config.kt

```diff
@@ -816,6 +816,26 @@ class Config : Vigilant(File("./config/skytils/config.toml"), "Skytils", sorting
     )
     var showGriffinCountdown = false
 
+    @Property(
+        type = PropertyType.SLIDER,
+        name = "Interval Between Burrow Refreshes",
+        description = "Amount of time to wait in between griffin burrow API refreshes.",
```

Comment:
There is no reason to allow the user to shorten the refresh cooldown, as it has no effect

Reply:
Oh does the API only refresh it only 60s, or something similar to that? I don't really know how the griffin API works :P